examples/init_setup_vars.yaml

```diff
@@ -0,0 +1,9 @@
+---
+
+## init-setup vars
+init_setup_enable: false
```

Comment:
The naming init_setup for the role and the variables seems misleading as we're only setting up a single repo here. We need to define it along the lines of openshift_tests_private_setup

Reply:
Changed the naming from `init_setup` to `openshift_tests_private_setup` for role variables.